packages/terra-table/src/TableCell.jsx

```diff
@@ -12,11 +12,14 @@ const propTypes = {
 const TableCell = ({
   content,
   ...customProps
-}) => (
-  <td {...customProps} data-class="cell">
-    {content}
-  </td>
-);
+}) => {
+  const dataClassNames = customProps['data-class'] ? `cell ${customProps['data-class']}` : 'cell';
```

Comment:
This seems like it adds an awkward level of maintenance and potential inconsistency throughout all of our modules we maintain. Is there enough value in keeping these data attributes solely for examining the dom structure? 

The consumer controls the css module classes through the css loader options in the webpack config. This is something some consumers might alter to significantly reduce the size of their assets. This will transform the css module classes into a unique hash, but leave data classes untransformed making the visual structure less valuable in the opposite direction.

![screen shot 2017-08-03 at 11 59 32 am](https://user-images.githubusercontent.com/6720991/28933598-3f24127c-7843-11e7-980a-4d4f6f7856a9.png)


If there's enough value in keeping it it might be easier to own a data attribute so we no longer need to merge potential attributes from custom props. Something like 'data-terra-class-identifier'